### PR TITLE
Makefile: add some podman

### PR DESCRIPTION
**Pull request by hdonnay**

### CODE REVIEW

**Comment by ldelossa** [comment]
@hdonnay would you be able to bring the http servers up and down as well ? Similar to the local-dev-up target?

**Comment by hdonnay** [comment]
Yeah, I can do that. I'll add that when I get the chance.

File: Makefile [hunk 1]
```diff
@@ -70,3 +70,59 @@ libscanhttp-restart:
...
+	podman create\
+		--pod claircore-dev\
+		--name claircore-database\
+		--env POSTGRES_USER=claircore\
+		--env POSTGRES_DB=claircore\
+		--env PGPORT=5434\
+		--expose 5434\
+		--volume $$(git rev-parse --show-toplevel)/internal/scanner/postgres/bootstrap.sql:/docker-entrypoint-initdb.d/libscan-bootstrap.sql:z\
+		--volume $$(git rev-parse --show-toplevel)/internal/vulnstore/postgres/bootstrap.sql:/docker-entrypoint-initdb.d/libvuln-bootstrap.sql:z\
+		--health-cmd "pg_isready -U claircore -d claircore"\
```
Comment (ldelossa) [inline Makefile]: My hunch is this is not going to block the other containers like 

```
	$(docker) exec -it claircore_claircore-db_1 bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
```

does - is that okay?

Comment (hdonnay) [reply Makefile]: You're right, I'll add a thing

**Pull request merged**